crt_portal/static/sass/custom/form.scss

```diff
@@ -359,3 +359,26 @@ select.usa-input--error {
     max-width: 80%;
   }
 }
+
+.field-required {
+  font-size: 1rem;
+  font-weight: normal;
+  font-style: italic;
+  color: $gray-50;
+
+  &::before {
+    content: "*";
+    font-weight: bold;
+    margin-left: 0.3rem;
+    color: $red-vivid-60;
```

Comment:
I would have preferred `@include u-text('red-60v');` but this doesn't give us the expected #B51D09 value (we get #B50909 instead)